### ISSUE TITLE
[Accessibility][Programmatic access] Contact 'Remove button' does not have name.

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11017]
Breakout of #6259

**Comment by joschect** [comment]
Remove button does not need a name, it is removable by pressing backspace and should not receive focus.
